gulpfile.js

```diff
@@ -55,9 +56,15 @@ gulp.task('tracks', function () {
              .pipe(gulp.dest('app/chrome/tracks'))
 })
 
+gulp.task('zip', ['copy'], function () {
```

Comment:
why do you need copy here again?

Reply:
I am establishing a dependency so the task 'zip' cannot start until all the tasks within 'copy' are done